mlflow/sklearn/__init__.py

```diff
@@ -740,17 +745,17 @@ def _log_pretraining_metadata(estimator, *args, **kwargs):  # pylint: disable=un
         # process; accordingly, we avoid logging initial, untuned parameters
         # for these seed estimators.
         should_log_params_deeply = not _is_parameter_search_estimator(estimator)
-        # Chunk model parameters to avoid hitting the log_batch API limit
-        for chunk in _chunk_dict(
-            estimator.get_params(deep=should_log_params_deeply),
-            chunk_size=MAX_PARAMS_TAGS_PER_BATCH,
-        ):
-            truncated = _truncate_dict(chunk, MAX_ENTITY_KEY_LENGTH, MAX_PARAM_VAL_LENGTH)
-            try_mlflow_log(mlflow.log_params, truncated)
```

Comment:
While not yet implemented, `AutologgingBatchingClient` will handle key / value truncation, chunking, and batching across params, tags, and metrics on a per-run basis.